libs/nest/config/src/lib/configurations/IdsClientConfig.ts

```diff
@@ -12,5 +12,9 @@ export const IdsClientConfig = defineConfig({
       '@island.is/clients/dev',
     ),
     clientSecret: env.required('IDENTITY_SERVER_CLIENT_SECRET'),
+    redirectUri: env.required(
```

Comment:
This should not be part of the `IdsClientConfig` object. You should add it to the `BaseTemplateAPIModuleConfig` as a 

```
userProfile: {
  redirectUri: string
}
```

then add it to `apps/application-system/api/src/environments/environment.ts`